incubator/cassandra/values.yaml

```diff
@@ -0,0 +1,44 @@
+## Cassandra image version
+## ref: https://hub.docker.com/r/library/cassandra/
+image:
+  repo: "cassandra"
```

Comment:
I try to use only official image to avoid maintenance in the future. What do you think?